packages/node/src/indexer/types.ts

```diff
@@ -1,7 +1,11 @@
 // Copyright 2020-2023 SubQuery Pte Ltd authors & contributors
 // SPDX-License-Identifier: GPL-3.0
 
-import { Entity } from '@subql/types-ethereum';
+import {
+  Entity,
+  EthereumBlock,
+  LightEthereumBlock,
+} from '@subql/types-ethereum';
 
 export enum OperationType {
```

Comment:
I think we no longer need these types, but maybe also could remove in branch `sync-20230829-eth`